components/brave_wallet_ui/components/desktop/asset-watchlist-item/style.ts

```diff
@@ -54,7 +54,8 @@ export const AssetIcon = styled.div<StyleProps>`
   width: 40px;
   height: 40px;
   border-radius: 100%;
-  background: ${(p) => p.icon ? `url(${p.icon})` : p.theme.color.background01}};
```

Comment:
@Douglashdaniel It seems there was a syntax error here (extra `}` in the end).

Reply:
Good catch!